fixwarn.js

```diff
@@ -0,0 +1,4 @@
+console.log('***************************************************')
+console.log('Make sure to run "npm run format" before commiting!')
+console.log('***************************************************')
+console.log(' ')
```

Comment:
Please just use `echo` in the NPM script. Using Node here is just extraneous.

Reply:
Whoops, will do :p

Reply:
Done in the latest commit